static/app/components/comboBox/comboBox.stories.tsx

```diff
@@ -89,6 +89,7 @@ export default storyBook('ComboBox', story => {
   });
 
   story('With growing input', () => {
+    const [value, setValue] = useState('opt_one');
```

Comment:
No value here meant that selecting an option in the story result in a weird flash and the combobox to not contain the clicked value